compiler/rustc_codegen_llvm/src/builder.rs

```diff
@@ -1232,12 +1356,6 @@ impl<'a, 'll, 'tcx> Builder<'a, 'll, 'tcx> {
         unsafe { llvm::LLVMGetBasicBlockParent(self.llbb()) }
     }
 
-    fn position_at_start(&mut self, llbb: &'ll BasicBlock) {
```

Comment:
I would prefer to flip back and forth between the two kinds of impls instead of moving the methods for now. Once we've stabilized what we can and want to move, we can bulk move things around.

Reply:
done.